default.nix

```diff
@@ -8,23 +8,32 @@
     overlays = [ ];
     inherit system;
   },
-  lib ? import "${sources.nixpkgs}/lib",
+  lib' ? import "${sources.nixpkgs}/lib",
 }:
 let
   dream2nix = (import sources.dream2nix).overrideInputs { inherit (sources) nixpkgs; };
   sops-nix = import "${sources.sops-nix}/modules/sops";
 in
 rec {
   inherit
-    lib
     pkgs
     system
     sources
     ;
 
-  # TODO: we should be exporting our custom functions as `lib`, but refactoring
-  # this to use `pkgs.lib` everywhere is a lot of movement
-  lib' = import ./lib.nix { inherit lib; };
+  lib = lib' // {
```

Comment:
Hm I don't think we should re-export *everything*, just our custom stuff

Reply:
Yes, something like that. Since we have only one (?) function in `lib'` that should be alright, and correctly signify we're doing something fancy.